handler/src/main/java/io/netty/handler/logging/ByteBufFormat.java

```diff
@@ -0,0 +1,20 @@
+package io.netty.handler.logging;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufUtil;
+
+/**
+ * Used to control the format and verbosity of {@link ByteBuf} logging.
```

Comment:
Isn't it also for `ByteBufHolder` logging ?

Reply:
Will fix.